src/styles/blocks/catalog.scss

```diff
@@ -11,15 +11,21 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition: all 0.5s;
```

Comment:
1. Move it to a line above the hover (+ indentation)
2. Use `transition` only for properties you transform  (`color` and `transform`)